web/impact/impact/graphql/query.py

```diff
@@ -0,0 +1,16 @@
+import graphene
+
+from impact.graphql.expert_profile import ExpertProfileType
+from accelerator.models.expert_profile import ExpertProfile
+
+
+class Query(graphene.ObjectType):
+    expert_profile = graphene.Field(ExpertProfileType, id=graphene.Int())
+
+    def resolve_expert_profile(self, info, **kwargs):
+        id = kwargs.get('id')
```

Comment:
`id` is a built-in python function, please rename

Reply:
@MattMSumner `expert_profile_id` would be fine